testdata/fixtures/group/descendant_services_response.json

```diff
@@ -5,7 +5,11 @@
         "descendantServices": {
           "nodes": [
             {
-              "id": "Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS8yNjE5"
+              "id": "Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS8yNjE5",
+              "aliases": [
```

Comment:
Just realized that I have to do this for all the responses corresponding to all the request queries that I updated. :/